### DISCUSSION TITLE
What happens after API has Responded

### DISCUSSION THREAD

**Discussion by KunalBurangi** [loopbackio/loopback-next#10502] [Q&A]
Hi All i have a question 
this is my jaeger trace 
![jaeger1](https://github.com/loopbackio/loopback-next/assets/37167732/af46f534-f48b-47a9-8667-eb9b049921fd)
if you see 1) is my API which has responded in 7-8 sec but the complete API took 12-13 sec 
I want to know what happens in  those remaining 5-6 sec
also what whatever is happening in between is it asynchronous?

**Comment by Salketer** [comment]
Can't say without more information... It seems you are grabbing lots of data, is there processing on the data after it is fetched from the database? Is it possible the serializing of the data take that much time? The 1 is not showing your API response time, more the mongoDB find time.
